src/common/components/ui/formik/PullSecretField.tsx

```diff
@@ -3,56 +3,71 @@ import { ExternalLinkAltIcon } from '@patternfly/react-icons';
 import { CLUSTER_MANAGER_SITE_LINK, PULL_SECRET_INFO_LINK } from '../../../config';
 import PopoverIcon from '../PopoverIcon';
 import TextAreaField from './TextAreaField';
+import { useTranslation } from '../../../hooks/use-translation-wrapper';
+// eslint-disable-next-line no-restricted-imports
```

Comment:
the disablement of eslint should not be needed.
 
please update eslint rule in  https://github.com/openshift-assisted/assisted-ui-lib/blob/master/.eslintrc.js#L67-L71

```
{
                name: 'react-i18next',
                importNames: ['useTranslation'],
                message:
                  'Please use useTranslation from src/common/hooks/use-translation-wrapper.ts instead',
              },
```